lib/src/utils.dart

```diff
@@ -15,10 +15,25 @@ bool isNumeric(int char) => char >= chars.zero && char <= chars.nine;
 
 /// Returns whether [path] has a URL-formatted Windows drive letter beginning at
 /// [index].
-bool isDriveLetter(String path, int index) {
-  if (path.length < index + 2) return false;
-  if (!isAlphabetic(path.codeUnitAt(index))) return false;
-  if (path.codeUnitAt(index + 1) != chars.colon) return false;
-  if (path.length == index + 2) return true;
-  return path.codeUnitAt(index + 2) == chars.slash;
+bool isDriveLetter(String path, int index) =>
+    driveLetterEnd(path, index) != null;
+
+/// Returns the index of the first character after the drive letter, or `null`
+/// if [index] is not the start of a drive letter.
+int? driveLetterEnd(String path, int index) {
```

Comment:
[nit] What do you think about the name `drivePathIndex`

Can the doc call out the specific behavior when `path` is exactly the drive letter with no following character?

Reply:
After re-reading the implementation, I think `driveLetterEnd` is a fine name.

Reply:
I've update the docs, let me know if this seems ok.